pkg/client/client_test.go

```diff
@@ -134,16 +169,25 @@ var _ = Describe("Client", func() {
 			close(done)
 		})
 
-		It("should create a new object from an unstructured object", func() {
+		It("should create a new object from an unstructured object", func(done Done) {
```

Comment:
Most of the time, our users are going to be using the core/custom types but unstructured. So not sure if unstructured test really helps ?

Reply:
Phil asked to do the unstructured, which gets slightly better code coverage. So let's go with it for now.